libbeat/docker-compose.yml

```diff
@@ -65,14 +65,14 @@ services:
       file: ${ES_BEATS}/testing/environments/${TESTING_ENVIRONMENT}.yml
       service: elasticsearch
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:9200"]
+      test: ["CMD-SHELL", "curl -s http://localhost:9200/_cluster/health | grep -vq '\"status\":\"red\"'"]
```

Comment:
Same approach as done in some other projects:
- https://github.com/elastic/opbeans-java/blob/master/docker-compose.yml#L91